web/modules/custom/helloworld/src/Controller/HelloController.php

```diff
@@ -0,0 +1,31 @@
+<?php 
+
+namespace Drupal\helloworld\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+
+/**
+ * Class for handling the routes of helloworld module.
+ */
+class HelloController extends ControllerBase {
+    /**
+   * Display the markup.
+   *
+   * @return array
+   *   Return markup array.
+   */
+  public function content() {
+    $user = \Drupal::currentUser();
```

Comment:
Using services like this in classes is not acceptable.
Please use dependency injection instead.